projects/hometown_homepage/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="main">
+      <!-- HERO SECTION -->
+      <div class="top">
+        <h1 class="visit">Visit Torshov</h1>
+        <h2 class="enjoy">
+          Enjoy culture and sports in this vibrant <br />
+          district close to the center of Oslo.
+        </h2>
```

Comment:
Somehow these elements are not centred both horizontally and vertically. Center them.